microsoft-edge/privacy-whitepaper.md

```diff
@@ -383,13 +395,12 @@ In Microsoft Edge, you can browse the web and translate webpages into a language
 
 Microsoft Edge allows you to install web apps made by website developers and pin your favorite sites.  
 
-Web apps are designed to create unique experiences for a website.  When you install a web app, the site for which it was installed and the app's unique name are stored locally on the device.  This data is not sent to Microsoft and is not synced across devices.  Your installed web apps are separated by profile.  Web apps use site permissions for the profile of which the app was installed.  You can manage your web apps by going to **edge://apps**.  These web apps may send info about how you use the web app and the sites you visit in the web app to Microsoft.  This diagnostic data collection is determined by your **Help improve Microsoft Edge** settings in Microsoft Edge Privacy and services settings.  To turn off this diagnostic data collection for Windows 10 Microsoft Edge Stable and Beta channels, go to Windows diagnostic data settings.  To turn off this diagnostic data collection for all other channels, go to **edge://settings/privacy** in a normal browsing session and turn off both **Improve Microsoft products by sending crash reports and data about how you use the browser** and **Make searches and Microsoft products better by sending info about websites you visit in Microsoft Edge**.  Clearing your browsing data from Microsoft Edge will clear your browsing data from web apps.   
 
 When you pin a site, it will be added to your taskbar or dock.  This data is stored locally on your device.  For some sites, info about whether the site has been pinned is shared with the site, so the site knows not to prompt to be pinned.  You can manage your pinned sites from the taskbar or dock.  Pinned sites open in Microsoft Edge windows and use the same site permissions and diagnostic data about how you use the browser and sites you visit as that version of Microsoft Edge.  
```

Comment:
```suggestion
When you pin a site, it will be added to your taskbar or dock.  This data is stored locally on your device.  For some sites, info about whether the site has been pinned is shared with the site, so the site knows not to prompt to be pinned.  You may manage your pinned sites from the taskbar or dock.  Pinned sites open in Microsoft Edge windows and use the same site permissions and diagnostic data about how you use the browser and sites you visit as that version of Microsoft Edge.  
```